dev/traffic_ops/seed.psql

```diff
@@ -94,6 +99,10 @@ INSERT INTO parameter (
 	'rascal.properties',
 	'health.polling.format',
 	'stats_over_http'
+), (
+	'CRConfig.json',
+	'geolocation.polling.url',
```

Comment:
Still need `coveragezone.polling.url` in order for TR to accept and save the CRConfig